src/Compilers/VisualBasic/Test/Syntax/Syntax/SyntaxTreeTests.vb

```diff
@@ -10,16 +10,25 @@ Imports Roslyn.Test.Utilities.TestHelpers
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
     Public Class VisualBasicSyntaxTreeTests
-        ' Diagnostic options on syntax trees are now obsolete
-#Disable warning BC40000
         <Fact>
-        Public Sub CreateTreeWithDiagnosticOptions()
+        Public Sub Create()
+            Dim root = SyntaxFactory.ParseCompilationUnit("")
+
+            Dim tree = VisualBasicSyntaxTree.Create(root)
+            Assert.Equal(SourceHashAlgorithm.Sha1, tree.GetText().ChecksumAlgorithm)
+
+            tree = VisualBasicSyntaxTree.Create(root, checksumAlgorithm:=SourceHashAlgorithms.Default)
+            Assert.Equal(SourceHashAlgorithms.Default, tree.GetText().ChecksumAlgorithm)
+        End Sub
+
+        <Fact, Obsolete>
```

Comment:
Ditto comment made about C# -- not sure I like the use of Obsolete here.

Reply:
I can add a string argument to Obsolete to make it clear that this is testing obsolete API.
Like here https://github.com/dotnet/roslyn/blob/main/src/Workspaces/CoreTest/Simplifier/SimplifierTests.cs#L45